components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tabletop from 'tabletop';
```

Comment:
What do you think about using `papaparse` like the [`tabletop` docs suggest](https://github.com/jsoma/tabletop#but-wait-dont-use-tabletop)? We already have `papaparse` as a dependency.

Reply:
Oh nice, that should work fine then, is it a simple swap?

Reply:
Good question! [The docs](https://github.com/jsoma/tabletop#but-wait-dont-use-tabletop) make it seem like it's not super involved to transition from `tabletop` to `papaparse`, but we'd need to give it a try to know for sure.